### DISCUSSION TITLE
Ability to add touch and go

### DISCUSSION THREAD

**Discussion by havber** [albar965/littlenavmap#887] [Ideas]
Hi, and thanks for your awesome contribution to the flight sim community!

I do a lot of VFR GA flying, and was thinking it would be cool if it was possible to "add touch and go" to an airport en route, even when that airport is not the destination of the flight plan. And if so, the TOD prior to the touch and go, and TOC after the touch and go could be calculated. 

Apologies if this is already possible without my knowledge, or if it is possible to achieve the same in ways that I am not aware of.

Thanks again!

**Comment by albar965** [comment]
Sorry, this is not possible. You have to create two flight plans for this. 
LNM allows only a flight plan with departure and destination.

**Reply by havber** [reply]
Ok, thanks for your swift reply!
